torsiondrive/td_api.py

```diff
@@ -158,7 +158,7 @@ def get_next_jobs(current_state, verbose=False):
     init_coords_M.xyzs = current_state['init_coords']
     init_coords_M.build_topology()
     # create a new scanner object
-    scanner = DihedralScanRepeater(QMEngine(), dihedrals, grid_spacing, init_coords_M, verbose)
+    scanner = DihedralScanRepeater(QMEngine(), dihedrals, grid_spacing, init_coords_M=init_coords_M, verbose=verbose)
```

Comment:
This is a bug fix. The bug uses `verbose` value `True` as values of other parameters like `energy_decrease_threshold` which will cause issue.